Controller/ConnectController.php

```diff
@@ -91,14 +94,14 @@ public function registrationAction(Request $request, $key)
             throw new NotFoundHttpException();
         }
 
-        $hasUser = $this->container->get('security.context')->isGranted('IS_AUTHENTICATED_REMEMBERED');
+        $hasUser = $this->container->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_REMEMBERED');
         if ($hasUser) {
             throw new AccessDeniedException('Cannot connect already registered account.');
         }
 
         $session = $request->getSession();
-        $error = $session->get('_hwi_oauth.registration_error.'.$key);
-        $session->remove('_hwi_oauth.registration_error.'.$key);
+        $error   = $session->get('_hwi_oauth.registration_error.' . $key);
```

Comment:
As I can see, there is no single coding standart here.

Here contiguous assignments are not aligned
https://github.com/hwi/HWIOAuthBundle/blob/master/Controller/ConnectController.php#L246-L247

And here they are
https://github.com/hwi/HWIOAuthBundle/blob/master/OAuth/ResourceOwner/AbstractResourceOwner.php#L77-L80

Is there any guide I can follow for this bundle?